codec/encoder/core/src/encoder_ext.cpp

```diff
@@ -3907,6 +3907,44 @@ int32_t WelsEncoderEncodeExt (sWelsEncCtx* pCtx, SFrameBSInfo* pFbi, const SSour
         WelsLog (pLogCtx, WELS_LOG_WARNING,
                  "WelsEncoderEncodeExt()MinCr Checking,codec bitstream size is larger than Level limitation");
     }
+
+    if (iSpatialNum == 1 && pFbi->bSaveRecon && NULL != pFbi->pReconData[0]) {
```

Comment:
We must check that pReconData has been correctly initialized or allocate memory for the recon frame otherwise.